<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>615</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>392</width>
    <height>615</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>392</width>
    <height>615</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 100</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>392</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="start">
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>390</y>
       <width>391</width>
       <height>82</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Learn">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255,255,255);
background-color: rgba(167, 187, 222, 150);
border-radius: 10</string>
        </property>
        <property name="text">
         <string>Learn</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Review">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255,255,255);
background-color: rgba(167, 187, 222, 150);
border-radius: 10</string>
        </property>
        <property name="text">
         <string>Review</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>371</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="settingButton">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 25px;
background-color: rgba(255,255,255,0);
color:rgb(85, 86, 87);
border-radius: 10</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bookButton">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 25px;
background-color: rgba(255,255,255,0);
color:rgb(85, 86, 87);
border-radius: 10</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="statisticButton">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 22px;
background-color: rgba(255,255,255,0);
color:rgb(85, 86, 87);
border-radius: 10</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="date">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>42</pointsize>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(201, 201, 201, 170);
border-radius: 10
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="startTitle">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>250</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>42</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(208, 153, 159, 0);
border-radius: 0</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#615efc;&quot;&gt;WordHen!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="setting"/>
   <widget class="QWidget" name="book">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>正在学习</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bookListButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>91</width>
       <height>131</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,255,255);
background-color: rgba(78, 78, 78, 120);
border-radius: 10;</string>
     </property>
     <property name="text">
      <string>book</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>150</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar{height:22px; text-align:center; font-size:14px; color:white; border-radius:7 px; background:#1D5573;}
QProgressBar::chunk{border-radius:11px;background:qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:0,stop:0 #99ffff,stop:1 #9900ff);}
</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>392</width>
     <height>615</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 50, 77);
color: rgba(208, 153, 159, 250);
border-radius: 0;
font-size: 60px;
font-weight: 900</string>
   </property>
   <property name="text">
    <string>WordHen!</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="winTitleBar">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: #EB5353;
	background-color: #EB5353;
	border-radius:6px;
}
QPushButton:hover {
	font-size:7px;
	color: rgb(100, 35, 35);
	background-color: #EB5353;
	border-radius:6px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="hideButton">
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: #F9D923;
	background-color: #F9D923;
	border-radius:6px;
}
QPushButton:hover {
	font-size:7px;
	color: rgb(100, 35, 35);
	background-color: #F9D923;
	border-radius:6px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="moveButton">
      <property name="minimumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox{
	color: #36AE7C;
	background-color: #36AE7C;
	border-radius:6px;
}
QCheckBox:hover{
	font-size:7px;
	background-color: rgb(37, 228, 123);
	border-radius:0px;
}
QCheckBox::indicator{/*选择框尺寸*/
    background-color: rgba(255, 255, 255, 0);
    border: 0px solid #b1b1b1;
	width: 12px;
	height: 12px;
}
QCheckBox::indicator:checked:hover{
	font-size:7px;
	color: rgb(100, 35, 35);
	background-color: #2789e4;
	border-radius:6px;
}
QCheckBox::indicator:!checked:hover{
	font-size:7px;
	color: rgb(100, 35, 35);
	background-color: #2789e4;
	border-radius:6px;
}
QCheckBox:enabled:checked{
	color: #37e49c;
	background-color: #37e49c;
	border-radius:6px;
}
QCheckBox:enabled:!checked{
	color: #36AE7C;
	background-color: #36AE7C;
	border-radius:6px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>hideButton</zorder>
   <zorder>moveButton</zorder>
   <zorder>closeButton</zorder>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="titleBarIcon">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(170, 170, 170); font-size:16px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="startTitle_2">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>42</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(208, 153, 159, 0);
border-radius: 0</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;WordHen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="shiftWallpaperButton">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: #615EFC;
	background-color: #615EFC;
	border-radius:8px;
}
QPushButton:hover {
	font-size:12px;
	color: rgb(100, 35, 35);
	background-color: #327ead;
	border-radius:8px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="startBG_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 127);
border-radius: 40;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/new/picture/pic/0.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="startBG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);
border-radius:0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>startButton</zorder>
  <zorder>startBG_2</zorder>
  <zorder>startBG</zorder>
  <zorder>widget</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>horizontalLayoutWidget_3</zorder>
  <zorder>horizontalLayoutWidget_4</zorder>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
